Meadowlark-js/package.json

```diff
@@ -60,7 +60,7 @@
     "test:e2e:build": "npm run docker:build && npm run test:e2e:jest",
     "test:e2e:build:wsl1": "npm run docker:build:wsl1 && npm run test:e2e:jest",
     "test:e2e:dev:setup": "cross-env TESTCONTAINERS_RYUK_DISABLED=true ts-node tests/e2e/setup/DevModeConfig.ts",
-    "test:e2e:dev:exit": "docker stop mongo-test opensearch-test meadowlark-api-test && docker rm -f mongo-test opensearch-test meadowlark-api-test && docker network prune -f",
+    "test:e2e:dev:exit": "bash ../eng/exit-dev-containers.sh",
```

Comment:
I'm curious, does this work for you in Windows? I assume you have WSL2. I still have WSL (1), can't remember why but something "broke" with 2. Running this command from PowerShell, bash just hangs and I have to kill it manually.

What do you think about changing to a PowerShell script, since PowerShell is available in GitHub Actions?

Reply:
We can do it in PowerShell. This script is not required to be executed on GitHub Actions, this is only a convenience for local development so there's no issue on changing to PS

Reply:
@stephenfuqua we have the [init-docker-compose.sh](https://github.com/Ed-Fi-Exchange-OSS/Meadowlark/blob/main/eng/init-docker-compose.sh) script where we handle wsl1 and wsl2, would something like this work for you?

Or do you prefer to have it as a PS script? I'm okay either way since we currently have a combination of both in the repo.

Reply:
It still just hangs on my computer, with no messages, and I have to kill it from the Task Manager (Control-C doesn't help). Since this is primarily a local command, and locally we're primarily running Windows, please change to PowerShell.